internal/render/helm.go

```diff
@@ -279,7 +281,9 @@ func makeStack(appname string, targetDir string, data []byte, stackVersion strin
 	if err != nil {
 		return errors.Wrap(err, "failed to load template compose")
 	}
-	os.Mkdir(filepath.Join(targetDir, "templates"), 0755)
+	if err := os.Mkdir(filepath.Join(targetDir, "templates"), 0755); err != nil {
+		return err
+	}
```

Comment:
This one needs checking for EEXISTS and not err out in that case, the helm command supports updating an existing chart.